packages/labs/preact-signals/package.json

```diff
@@ -0,0 +1,141 @@
+{
+  "name": "@lit-labs/preact-signals",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Preact Signals integration for Lit",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit.dev/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/lit/lit.git",
+    "directory": "packages/labs/preact-signals"
+  },
+  "type": "module",
+  "main": "index.js",
+  "module": "index.js",
+  "typings": "index.d.ts",
+  "directories": {
+    "test": "test"
+  },
+  "exports": {
+    ".": {
+      "types": "./development/index.d.ts",
+      "development": "./development/index.js",
+      "default": "./index.js"
+    }
+  },
+  "files": [
+    "/development/",
+    "!/development/test/",
+    "/index.{d.ts,d.ts.map,js,js.map}",
+    "/lib/"
+  ],
+  "scripts": {
+    "build": "wireit",
+    "build:ts": "wireit",
+    "build:ts:types": "wireit",
+    "test": "wireit",
+    "test:dev": "wireit",
+    "test:prod": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "dependencies": [
+        "build:rollup",
+        "build:ts",
+        "build:ts:types",
+        "../../lit:build"
+      ]
+    },
+    "build:ts": {
+      "command": "tsc --build --pretty",
+      "dependencies": [
+        "../../lit:build:ts:types"
+      ],
+      "clean": "if-file-deleted",
+      "files": [
+        "src/**/*.ts",
+        "src/**/*.tsx",
+        "tsconfig.json"
+      ],
+      "output": [
+        "development",
+        "tsconfig.tsbuildinfo"
+      ]
+    },
+    "build:ts:types": {
+      "command": "treemirror development . \"**/*.d.ts{,.map}\"",
+      "dependencies": [
+        "../../internal-scripts:build",
+        "build:ts"
+      ],
+      "files": [],
+      "output": [
+        "*.d.ts{,.map}",
+        "lib/*.d.ts{,.map}",
+        "test/**/*.d.ts{,.map}"
+      ]
+    },
+    "build:rollup": {
+      "command": "rollup -c",
+      "dependencies": [
+        "build:ts"
+      ],
+      "files": [
+        "rollup.config.js",
+        "../../rollup-common.js",
+        "src/test/*_test.html"
+      ],
+      "output": [
+        "index.js{,.map}",
+        "lib/*.js{,.map}",
+        "test/*_test.html"
+      ]
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "command": "MODE=dev node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
+      "dependencies": [
+        "build:ts",
+        "../../tests:build"
+      ],
+      "env": {
+        "BROWSERS": {
+          "external": true
+        }
+      },
+      "files": [],
+      "output": []
+    },
+    "test:prod": {
+      "command": "MODE=prod node ../../tests/run-web-tests.js \"development/**/*_test.js\" --config ../../tests/web-test-runner.config.js",
```

Comment:
Ah, probably not, this is handled by export conditions I think

Reply:
it should use development because the rollup build does not copy tests. We use export conditions to select the prod builds of the libraries under test.